client/etcd/client.go

```diff
@@ -109,10 +109,7 @@ func (c *csclient) KV() (kv.Store, error) {
 
 func (c *csclient) Txn() (kv.TxnStore, error) {
 	c.txnOnce.Do(func() {
-		kvOpts := kv.NewOptions().
-			SetNamespace(kvPrefix).
-			SetEnvironment(c.opts.Env())
-		c.txn, c.txnErr = c.createTxnStore(kvOpts)
+		c.txn, c.txnErr = c.TxnStore(kv.NewOptions())
```

Comment:
We don't need to set the namespace and environment here anymore?

Reply:
That's taken care of in `sanitizeOptions`.

Reply:
Oh, I see, Github had collapsed those lines in my view lol.